frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -361,9 +381,17 @@ pods:
         env:
           DATANODE: true
         # If HDFS does not report a data node to be live, we don't consider the data node to be ready yet
-        # TODO: LIBPROCESS_IP gets overriden to 0.0.0.0 in overlay networking so this readiness-check needs to be fixed
         readiness-check:
+          {{#KERBEROS_ENABLED}}
+          cmd: >
+            export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/) &&
```

Comment:
Any reason why we don't just add the `KERBEROS_ENABLED` section after the `JAVA_HOME` and before `TASK_IP`?

Reply:
Yup, we could do that.